src/di.php

```diff
@@ -258,13 +259,13 @@
 };
 
 /*
- *
  * @param void
  *
- * @return \FileCache
+ * @return \Symfony\Component\Cache\Adapter\FilesystemAdapter
  */
 $di['cache'] = function () {
-    return new FileCache();
+    // Reference: https://symfony.com/doc/current/components/cache/adapters/filesystem_adapter.html
+    return new FilesystemAdapter('', 0, PATH_CACHE . '/sf_cache');
```

Comment:
Couple of little things:
1. I'm not sure how other's feel about this, but I'd prefer it if we didn't set the default lifetime to `0`. That causes it to be kept indefinitely, and I'm not aware of any task we have to clear the cache except when a system update is performed. I'd much rather prefer the default lifetime be somewhere between 24 hours and a week.
2. I think we should be using the "namespace" option rather than adding a sub-dir to the path.
   a. Example: `return new FilesystemAdapter('sf_cache', 0, PATH_CACHE);`
   b. According to the Symfony docs, this option defines "a string used as the subdirectory of the root cache directory, where cache items will be stored"

Other than that, this looks great. Thanks!

Reply:
Makes sense. Thanks for the review.